selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -1032,8 +1032,10 @@ void process_road_camera(MultiCameraState *s, CameraState *c, int cnt) {
   if ((c == &s->road_cam && env_send_road) || (c == &s->wide_road_cam && env_send_wide_road)) {
     framed.setImage(get_frame_image(b));
   }
+  LOGT((std::string(c == &s->road_cam ? "RoadCamera" : "WideRoadCamera")+std::string(": Image set")).c_str());
```

Comment:
can you build these strings in a nicer way?

Reply:
Tried to use va_args to allow for an unknown number of c-str arguments, but got stuck